<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunConfigurations</class>
 <widget class="QDialog" name="RunConfigurations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run Configurations</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
    border: 1px solid black;
    border-radius: 9px;
    margin-top: 0.5em;
}
QGroupBox::title {
    subcontrol-position:top middle;
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
	color:rgb(255, 255, 255);
}
QLabel {
	color:rgb(255, 255, 255);
}
QLabel:disabled {
	color:rgb(120, 120, 120);
}
QCheckBox{
	color:rgb(255, 255, 255);
}
QCheckBox:disabled{
	color::rgb(120, 120, 120);
}
QLineEdit:disabled{
	background-color:rgb(180, 180, 180);
}
QWidget:disabled{
	background-color:rgb(180, 180, 180);
}
QTextEdit:disabled{
	background-color:rgb(180, 180, 180);
}
QComboBox:disabled{
	background-color:rgb(180, 180, 180);
}
QLineEdit:disabled{
	background-color:rgb(120, 120, 120);
}
background-color: rgb(80, 80, 80);
</string>
  </property>
  <widget class="QFrame" name="Run_Configurations_Frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 80, 80)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTabWidget" name="Run_Confirmations_Tab_Widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 50, 50);
border-radius: 5px;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Misc_Config_Tab">
     <attribute name="title">
      <string>Misc. Run Configurations</string>
     </attribute>
     <widget class="QGroupBox" name="Simulation_Options_GroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>296</width>
        <height>156</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;
</string>
      </property>
      <property name="title">
       <string>Simulation Options</string>
      </property>
      <widget class="QCheckBox" name="Wireless_CheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>20</y>
         <width>156</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Enable Wireless</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="Power_Saving_Mode_CheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Power Saving Mode</string>
       </property>
      </widget>
      <widget class="QWidget" name="Random_Number_Seed_Widget" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>281</width>
         <height>26</height>
        </rect>
       </property>
       <widget class="QLabel" name="Random_Number_Seed_Label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>156</width>
          <height>26</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 11pt;
font-style: italic;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Random Number Seed:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="Random_Number_Seed_SpinBox">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>0</y>
          <width>111</width>
          <height>27</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Verbose_Widget" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>281</width>
         <height>21</height>
        </rect>
       </property>
       <widget class="QComboBox" name="Verbose_ComboBox">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>0</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 0px;
text-align: center;</string>
        </property>
       </widget>
       <widget class="QLabel" name="Verbose_Label">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>0</y>
          <width>86</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 11pt;
font-style: italic;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Verbosity:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QCheckBox" name="Detailed_Output_Checkbox">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>65</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Detailed Output</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Administrative_GroupBox">
      <property name="geometry">
       <rect>
        <x>315</x>
        <y>10</y>
        <width>316</width>
        <height>156</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;
</string>
      </property>
      <property name="title">
       <string>Administrative</string>
      </property>
      <widget class="QCheckBox" name="Trace_Mode_CheckBox">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>20</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Trace Mode</string>
       </property>
      </widget>
      <widget class="QWidget" name="Trace_Mode_File_Widget" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>286</width>
         <height>21</height>
        </rect>
       </property>
       <widget class="QPushButton" name="Trace_Mode_File_PushButton">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>0</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(240, 240, 240, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>File...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Trace_Mode_File_LineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-1</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 0px;
font-size: 10pt;
</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Power_Configuration_GroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>296</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;
</string>
      </property>
      <property name="title">
       <string>Power Configuration</string>
      </property>
      <widget class="QListView" name="Power_Configuration_ListView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>276</width>
         <height>161</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 0px;
font-size: 10pt;
text-align: right;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Power_Configuration_Delete_PushButton">
       <property name="geometry">
        <rect>
         <x>135</x>
         <y>190</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(240, 240, 240, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Power_Configuration_Add_PushButton">
       <property name="geometry">
        <rect>
         <x>215</x>
         <y>190</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(240, 240, 240, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Power_Configuration_Edit_PushButton">
       <property name="geometry">
        <rect>
         <x>55</x>
         <y>190</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(240, 240, 240, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Edit...</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>405</x>
        <y>215</y>
        <width>146</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/assets/logo.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>230</y>
        <width>151</width>
        <height>106</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/assets/logo.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Sweeping_Config_Tab">
     <attribute name="title">
      <string>Sweeping</string>
     </attribute>
     <widget class="QGroupBox" name="Sweep_Options_GroupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>296</width>
        <height>186</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;
</string>
      </property>
      <property name="title">
       <string>Sweep Options</string>
      </property>
      <widget class="QWidget" name="Variable_Sweep_Widget" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>25</y>
         <width>271</width>
         <height>26</height>
        </rect>
       </property>
       <widget class="QComboBox" name="Variable_Sweep_ComboBox">
        <property name="geometry">
         <rect>
          <x>105</x>
          <y>0</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 0px;
text-align: center;</string>
        </property>
       </widget>
       <widget class="QLabel" name="Variable_Sweep_Label">
        <property name="geometry">
         <rect>
          <x>-5</x>
          <y>1</y>
          <width>106</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 11pt;
font-style: italic;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Variable:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Sweep_Output_Options_GroupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>315</x>
        <y>30</y>
        <width>316</width>
        <height>186</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;
</string>
      </property>
      <property name="title">
       <string>Output Options</string>
      </property>
      <widget class="QRadioButton" name="Sweep_Output_BarGraph_RadioButton">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>25</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Bar Graph</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="Sweep_Output_LineGraph_RadioButton">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>55</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Line Graph</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="Sweep_Output_ScatterPlot_RadioButton">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>85</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Scatter Plot</string>
       </property>
      </widget>
      <widget class="QGraphicsView" name="Sweep_Output_GraphicsView">
       <property name="geometry">
        <rect>
         <x>135</x>
         <y>20</y>
         <width>171</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QWidget" name="Sweep_Output_Image_Widget" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>296</width>
         <height>21</height>
        </rect>
       </property>
       <widget class="QPushButton" name="Sweep_Output_Image_PushButton">
        <property name="geometry">
         <rect>
          <x>225</x>
          <y>0</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(240, 240, 240, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>File...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Sweep_Output_Image_LineEdit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>211</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 0px;</string>
        </property>
       </widget>
      </widget>
      <widget class="QCheckBox" name="Sweep_Output_Image_CheckBox">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>120</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Copy to Image:</string>
       </property>
      </widget>
     </widget>
     <widget class="QCheckBox" name="Enable_Sweeping_CheckBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>5</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable Sweeping (still in development)</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QDialogButtonBox" name="Confirm_ButtonBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>445</y>
      <width>621</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(240, 240, 240, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
